Vision/Vision.bs

```diff
@@ -140,7 +140,8 @@ Markup Shorthands: markdown yes
 		organizational sizes,
 		and more.
 		In order to ensure W3C serves the needs of the entire Web user base, 
-		we also strive to broaden diversity and inclusion for our own participants.
+		we strive to actively include their participation
```

Comment:
```suggestion
		we strive broaden inclusion
```

Reply:
Thanks @cwilso I appreciate the repair attempt. Rather than rushing a last minute (<24h to AB telcon) fix to this PR (which has not been labeled as a blocker) before publishing an updated NOTE, I'd prefer (and believe it is more appropriate) for us to take time to address it within the context of the issue, explicitly including the non-AB participants in the discussion both in this PR and issue #169.